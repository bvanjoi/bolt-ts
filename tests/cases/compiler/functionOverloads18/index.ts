// From `github.com/microsoft/TypeScript/blob/v5.8.2/tests/cases/compiler/functionOverloads18.ts`, Apache-2.0 License

function foo(bar:{a:number;});
//~^ ERROR: This overload signature is not compatible with its implementation signature.
function foo(bar:{a:string;}) { return {a:""} }
