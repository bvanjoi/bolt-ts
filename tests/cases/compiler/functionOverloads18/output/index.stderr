  × This overload signature is not compatible with its implementation
  │ signature.
   ╭─[../../tests/cases/compiler/functionOverloads18/index.ts:3:10]
 3 │ foo
   · ───
   ╰────
